app/components/global-error/global-error.tsx

```diff
@@ -27,8 +32,20 @@ export const GlobalErrorToast: ComponentType = () => {
   }
 
   if (networkError.statusCode >= 400 && networkError.statusCode < 500) {
-    // TODO translation
-    toastShow("Request issue.\nContact support if the problem persists")
+    const errorCode = (networkError as ServerError).result?.errors?.[0]?.code
+    switch (errorCode) {
+      case NetworkErrorCode.InvalidAuthentication:
+        toastShow(translate("common.loggedOut"), {
+          duration: Toast.durations.SHORT,
+          onHidden: () => logout(),
+        })
+        break
+
+      default:
+        // TODO translation
+        toastShow("Request issue.\nContact support if the problem persists")
```

Comment:
Sorry I know this isn't your code and you have just moved it - but do you mind moving this error into the translation files?

Reply:
Sure, np, I just didn't know how far I could go with the changes, I just pushed a commit with all the GlobalError messages translated